tests/test_operation.py

```diff
@@ -196,6 +194,54 @@ def test_operation_inside_context_do_queue_false(self, qnode):
         assert qnode.ops[2].name == "PauliX"
         assert qnode.ops[3].name == "PauliZ"
 
+    @pytest.fixture(scope="function")
+    def qnode_for_inverse(self, mock_device):
+        """Provides a QNode for the subsequent tests of inv"""
+
+        def circuit(x):
+            qml.RZ(x, wires=[1]).inv()
+            qml.RZ(x, wires=[1]).inv().inv()
+            return qml.expval(qml.PauliX(0)), qml.expval(qml.PauliZ(1))
+
+        node = qml.QNode(circuit, mock_device)
+        node.construct([1.0])
+
+        return node
+
+    def test_operation_inverse_defined(self, qnode_for_inverse):
+        """Test that the inverse of an operation is added to the QNode queue and the operation is an instance
+        of the original class"""
+        assert qnode_for_inverse.ops[0].name == "RZ" + qml.operation.Operation.string_for_inverse
```

Comment:
Can we be explicit here, instead of using `qml.operation.Operation.string_for_inverse`?

Reply:
Sure, changed it